audmetric/core/api.py

```diff
@@ -1186,28 +1186,68 @@ def weighted_confusion_error(
 def word_error_rate(
     truth: Sequence[Sequence[str]],
     prediction: Sequence[Sequence[str]],
+    *,
+    norm: str = "truth",
 ) -> float:
     r"""Word error rate based on edit distance.
 
+    The word error rate is computed by aggregating the normalized edit
+    distances of each (truth, prediction)-pair and averaging the
+    aggregated score by the number of pairs.
+
+    The normalized edit distance of each (truth, prediction)-pair is computed
+    as the edit distance divided by a normalization factor n. This represents
+    the average editing cost per sequence item.
+
+    The value of n depends on the norm parameter:
+    If ``norm`` is ``"truth"`` (default):
+    n is set to the reference (truth) length, following the Wikipedia formulation
+    where n is the number of words in the reference (N=S+D+C). This means WER can
+    be greater than 1 if the prediction sequence is longer than the reference:
```

Comment:
And some more semantic line breaks:

```suggestion
    n is set to the reference (truth) length,
    following the Wikipedia formulation
    where n is the number of words in the reference (N=S+D+C).
    This means WER can be greater than 1
    if the prediction sequence is longer than the reference:
```

Reply:
```
following the Wikipedia formulation
where n is the number of words in the reference (N=S+D+C).
```

I would either skip  `(N=S+D+C)` or say what `S, D, and C` stands for.

Reply:
There was a second concern about the overcomplex notation: I have removed S,D, and C entirely, rephrased a bit, and added what I consider (after trying to understand the information in the link) semantic line breaks.

I have therefor not committed this suggestion, the actual commit being a mix of SLB and notion concerns.

Reply:
Thanks. I think your updated docstring looks fine.